src/components/button/test-button.js

```diff
@@ -30,6 +30,43 @@ describe('Button', function() {
     };
   });
 
+  it('default matches the snapshot', function() {
+    const wrapper = renderComponent();
+    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.find('.p-button').length).toEqual(1);
```

Comment:
Nice, I like this approach, using the snapshot and being explicit about what is being checked.